wrappers/services.go

```diff
@@ -191,43 +191,32 @@ func startUserServices(cli *client.Client, inter Interacter, services ...string)
 }
 
 func stopService(sysd systemd.Systemd, app *snap.AppInfo, inter Interacter) error {
-	serviceName := app.ServiceName()
-	tout := serviceStopTimeout(app)
+	var serviceList []string
 
-	var extraServices []string
+	// Add application sockets
 	for _, socket := range app.Sockets {
-		extraServices = append(extraServices, filepath.Base(socket.File()))
+		serviceList = append(serviceList, filepath.Base(socket.File()))
 	}
+	// Add application timers
 	if app.Timer != nil {
-		extraServices = append(extraServices, filepath.Base(app.Timer.File()))
+		serviceList = append(serviceList, filepath.Base(app.Timer.File()))
 	}
+	// Add application service
+	serviceList = append(serviceList, app.ServiceName())
 
 	switch app.DaemonScope {
 	case snap.SystemDaemon:
-		var stopErrors error
-		if len(extraServices) > 0 {
-			stopErrors = sysd.Stop(extraServices, tout)
-		}
-
-		if err := sysd.Stop([]string{serviceName}, tout); err != nil {
-			if !systemd.IsTimeout(err) {
-				return err
-			}
-			inter.Notify(fmt.Sprintf("%s refused to stop, killing.", serviceName))
-			// ignore errors for kill; nothing we'd do differently at this point
-			sysd.Kill(serviceName, "TERM", "")
```

Comment:
yes, though it might have not done so consistently in some old version (but which?). But for example the new user session daemon code  dropped this bit as well